lib/config.js

```diff
@@ -81,6 +81,7 @@ var ENV_TABLE = {
   abortedErrorThreshold: 'ELASTIC_APM_ABORTED_ERROR_THRESHOLD',
   active: 'ELASTIC_APM_ACTIVE',
   addPatch: 'ELASTIC_APM_ADD_PATCH',
+  apiKey: 'ELASTIC_APM_API_KEY',
   apiRequestSize: 'ELASTIC_APM_API_REQUEST_SIZE',
```

Comment:
👍 I see here that we're adding the ENV variable `ELASTIC_APM_API_KEY`, which matches what the other agent teams have done and is the [agreed upon spec behavior](https://github.com/elastic/apm/issues/183).